src/bento.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @fileoverview
+ * Entry point for `bento.js`
+ */
+
+import {install as installCustomElements} from './polyfills/custom-elements';
+
+installCustomElements(self);
+
+const BENTO = self.BENTO || [];
+
+BENTO.push = (fn) => {
+  fn();
+};
+
+// eslint-disable-next-line local/window-property-name
+self.BENTO = BENTO;
+
+for (let i = 0; i < BENTO.length; i++) {
+  BENTO.push(BENTO[i]);
```

Comment:
Any reason we use `.push` here instead of just...calling the function? Do we expect `push()` to acquire more functionality?

```suggestion
  const install = BENTO[I];
  install();
```

Reply:
It would be nice if the `init` tasks are deferred until a Bento element registers, which is why I'm calling `push`. Don't feel strongly either way.